### PR TITLE
add docs for view integration

### PR DESCRIPTION
**Pull request by bglusman**
@devmanhinton give this a look see

### CODE REVIEW

File: docs/view-integration.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+View Integration
+================
+
+Though EndDash let's you avoid having to create Backbone Views for many pieces
+of dynamic behavior in your application by putting view logic in models/presenters
+and changing the view state based on their state directly, it also helps simplify
+configuring and initializing them when you need view logic.
```
Comment (devmanhinton) [inline docs/view-integration.md:7]: This looks all really good. This top portion might be a bit wordy :D

Comment (bglusman) [inline docs/view-integration.md:7]: Feel free to edit or suggest how to trim down... I'm sending this as is with other docs to Freddie as a draft to style for now...
